templates/CRM/common/navigation.js.tpl

```diff
@@ -54,30 +54,18 @@
 {/strip}{/capture}// <script> Generated {$smarty.now|date_format:'%d %b %Y %H:%M:%S'}
 {literal}
 (function($) {
-  var menuMarkup = {/literal}{$menuMarkup|@json_encode};
-{if $config->userFramework neq 'Joomla'}{literal}
-  $('body').append(menuMarkup);
+  var menuMarkup = {/literal}{$menuMarkup|@json_encode}{literal};
 
-  $('#civicrm-menu').css({position: "fixed", top: "0px"});
-
-  //Track Scrolling
-  $(window).scroll(function () {
-    $('div.sticky-header').css({top: "23px", position: "fixed"});
-  });
```

Comment:
It looks like this is being removed completely, but I don't think it should be. I was mistaken that it is a drupal thing. See #12536

Reply:
Also this change is now causing a merge-conflict. I suggest rebasing it out as this line shouldn't be getting removed.